modules/xcode/_preload.lua

```diff
@@ -69,7 +69,7 @@
 
 		-- Xcode always uses Mac OS X path and naming conventions
 
-		toolset  = "clang",
+		toolset  = "clang", -- TODO: os.target() == p.MACOSX and p.checkVersion(minOSVersion, "<10.7") -> gcc
```

Comment:
I don't know how to set it here (no `cfg` for `systemversion`)...

Reply:
I'm fine with leaving this as an open question for now.